graphlearn/service/dist/grpc_channel.cc

```diff
@@ -92,6 +96,7 @@ Status GrpcChannel::CallStop(const StopRequestPb* req, StopResponsePb* res) {
   ::grpc::ClientContext ctx;
   SetContext(&ctx);
   ::grpc::Status s = stub_->HandleStop(&ctx, *req, res);
+  stoped = true;
```

Comment:
Set `stoped=true` at the begining of `CallStop`, to avoid concurrent contention.

Reply:
fixed.